### DISCUSSION TITLE
Option to add a clean container without junk html

### DISCUSSION THREAD

**Discussion by speedxcz** [elementor/elementor#28186] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Hi, very often I run into the problem that any element added to the page contains an extreme amount of junk html css code. I want to add an H3 heading to an already existing structure, immediately there is an extra 1kb of junk html. I want to insert a shortcode into the loop template, immediately 1kb (times the number of repetitions) of html is added to it, which is not used for anything.
It would be possible to add a container, feel free to call it expert container or super container. Feel free to have its own power button with a warning of its own danger in the Elemntor settings. Above all, such a container should have no or only minimal html code <div></div>. I think this would take the builder to a completely different level.

### Describe the solution you'd like.

This is it:
`<div>Custom content shortcode</div>`

Instead of the existing one:

```
<div data-elementor-type="loop-item" data-elementor-id="35878" class="elementor elementor-35878 e-loop-item e-loop-item-36745 post-36745 post type-post status-publish format-standard has-post-thumbnail hentry category-informace tag-akce tag-architekti tag-interier tag-jap-future tag-partnerske-clanky" data-elementor-post-type="elementor_library" data-custom-edit-handle="1">
			<div class="elementor-element elementor-element-8d829ee e-con-full e-flex e-con e-parent e-lazyloaded" data-id="8d829ee" data-element_type="container">
				<div class="elementor-element elementor-element-b94fe48 elementor-widget elementor-widget-shortcode" data-id="b94fe48" data-element_type="widget" data-widget_type="shortcode.default">
				<div class="elementor-widget-container">
					<div class="elementor-shortcode">Custom content shortcode</div>
				</div>
				</div>
				</div>
				</div>
```

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by dimitris-mourtzis** [comment]
I am also looking forward to this feature!!!
